internal/exec/terraform.go

```diff
@@ -181,7 +181,7 @@ func ExecuteTerraform(cmd *cobra.Command, args []string) error {
 	// Handle `terraform deploy` custom command
 	if info.SubCommand == "deploy" {
 		info.SubCommand = "apply"
-		if !info.UseTerraformPlan && !u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag) {
+		if !(info.UseTerraformPlan && u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag)) {
```

Comment:
this changes the logic

https://brilliant.org/wiki/de-morgans-laws/

```
Not (A and B) is the same as Not A or Not B
```

`!info.UseTerraformPlan && !u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag)` will be true id both are false

`!(info.UseTerraformPlan && u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag))` will be true if ANY is false

Reply:
You are absolutely right...my bad for not catching that!

1ea35bec90028a8b5928c0f3b809bda4766caeb9